config/mainCfg_ETau_ttCR_Legacy2017.cfg

```diff
@@ -0,0 +1,83 @@
+[general]
+
+lumi = 41529 # pb^-1
+# the lumi in fb^-1 is used only by the plotting program
+# approximate to only one decimal digit
+lumi_fb = 41.5 # fb^-1
+
+outputFolder = /home/llr/cms/motta/CMSSW_11_1_0_pre6/src/KLUBAnalysis/MY_ttCRlimits_2017inclusive/ETau/
+
+data = DsingleEleB, DsingleEleC, DsingleEleD, DsingleEleE, DsingleEleF
+
+signals = GGHH_NLO_cHHH1_xs
+
+backgrounds = TTfullyHad, TTfullyLep, TTsemiLep, DY_0b_1Pt, DY_0b_2Pt, DY_0b_3Pt, DY_0b_4Pt, DY_0b_5Pt, DY_0b_6Pt, DY_1b_1Pt, DY_1b_2Pt, DY_1b_3Pt, DY_1b_4Pt, DY_1b_5Pt, DY_1b_6Pt, DY_2b_1Pt, DY_2b_2Pt, DY_2b_3Pt, DY_2b_4Pt, DY_2b_5Pt, DY_2b_6Pt, DYJets_M_10_50_Not_PU_Safe, DYJets_M_10_50_PU_Safe, WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf, TWtop, TWantitop, singleTop_top, singleTop_antitop, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, ZH_HBB_ZLL, ZH_HBB_ZQQ, ZH_HTauTau, VBFHTauTau, ggHTauTau, WplusHTauTau, WminusHTauTau, ZZTo4L, ZZTo2L2Nu, ZZTo2L2Q, ZZTo2Q2Nu, WZTo3LNu, WZTo1L1Nu2Q, WZTo1L3Nu, WZTo2L2Q, WWTo2L2Nu, WWToLNuQQ, WWTo4Q, ZZZ, WZZ, WWW, WWZ, ttHJetToBB, ttHJetTononBB, ttHToTauTau, TTWJetsToLNu, TTWJetsToQQ, TTZZ, TTWW, TTWZ, TTWH, TTZH, TTZToLLNuNu, TTZToQQ
+
+variables = bH_mass
```

Comment:
This should be `DNNoutSM_kl_1`

Reply:
fixed

Reply:
This is still `bH_mass` instead of `DNNoutSM_kl_1`